packages/shared/notionable-hooks/src/use-wallet.ts

```diff
@@ -49,34 +50,34 @@ export function useWalletConnected() {
 
 export function useWalletAddress() {
   const {
-    globalState: { wallet },
-  } = useNotionalContext();
+    wallet: { userWallet },
+  } = useAppStore();
 
-  return wallet?.selectedAddress;
+  return userWallet?.selectedAddress;
```

Comment:
Should these be wrapped in useObserver? or is that already done in useAppStore?

Reply:
I am hesitant to throw useObserver around in the hooks. I am in favor of using observer at the component level. I think it makes it more obvious what is supposed to be rendering and what is not. It'd be easy to forget which hooks use useObserver and which do not.

Reply:
ok thats fine